src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -175,7 +201,8 @@ InModuleScope Arcus.Scripting.IntegrationAccount {
                         
                         $actual = Get-AzIntegrationAccountSchema -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemaName $expectedSchemaName
                         $actual | Should -Not -BeNullOrEmpty
-                        $actual.CreatedTime.ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToString("yyyy-MM-ddTHH:mm:ss")
+                        $actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -BeIn ($actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss"), $actual.ChangedTime.ToUniversalTime().AddSeconds(-1).ToString("yyyy-MM-ddTHH:mm:ss"))
+                        #$actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss")
                         $actual.CreatedTime | Should -BeGreaterOrEqual $executionDateTime
```

Comment:
```suggestion
                        $actual.CreatedTime | Should -BeGreaterOrEqual $executionDateTime
```

Can be removed?

Reply:
Argh, yes..
Should've checked and removed these comments before committing 😅